File: TCS_NQT_2026/TCS_MY_SHIFT_QUE/odd_ticket.java
package TCS_NQT_2026.TCS_MY_SHIFT_QUE;
import java.util.*;
public class odd_ticket {
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int [] arr = new int[n];
        for (int i = 0; i<n;i++){
            arr[i] = sc.nextInt();
        }
        int sum_odd = 0;
        int count = 0;
        for(int num:arr){
            if(num % 2 != 0){
                sum_odd += num;
                count += 1;
            }
        }
        double avg;
        if (count > 0){
            avg = (double) sum_odd / count;
        }
        else{
            avg = 0;
        }
        System.out.println(sum_odd + " " +count +" " + avg);

    }
}

File: TCS_NQT_2026/TCS_MY_SHIFT_QUE/odd_ticket.py
n = int(input())
arr  = list(map(int,input().split()))

sum_odd = 0
count  = 0
for num in arr:
    if num % 2 != 0:
        sum_odd += num
        count += 1

# instead of this code i write this code....
# avg = sum_odd / count ..... that's why the error are occur..
if count > 0:
    avg = sum_odd / count
else:
    avg = 0

print(sum_odd,count,avg)
